doc/api/module.md

```diff
@@ -80,6 +80,96 @@ isBuiltin('fs'); // true
 isBuiltin('wss'); // false
 ```
 
+### `module.registerLoader()`
+
+<!-- YAML
+added: v20.x.x
+-->
+
+In addition to using the `--experimental-loader` option in the CLI,
+loaders can be registered programmatically using the `module.registerLoader()`
+method.
+
+```js
+import { registerLoader } from 'node:module';
+
+registerLoader('http-to-https');
+
+// Loader hooks haven't been executed yet.
```

Comment:
I don't understand this comment

Reply:
What I mean with this comment is that if you call `registerLoader()`, your loader will only be available in the next imported module, meaning that the hooks of your loader will not be executed until `import` statement call, how I could make it clearer?

Reply:
Ahh. I think the comment is more confusing than not saying anything. Other documentation already explains this better :)